.circleci/scripts/publish_android_snapshot.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+set -eu -o pipefail
```

Comment:
This file handles credentials, it's probably a good idea to warn people not to put `-x` in this file. (Although that's probably less important now that CircleCI has gotten secret masking.)

Reply:
Yes, I will add the comment, with -x all secret environment variables will be shown as there is a `if [ -z` checks for them and '-x' will print arguments :)